src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/EvidenceNoteHistoryDataViewModelMapTransfer.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.Mappings.ToViewModels
+{
+    using EA.Weee.Core.AatfEvidence;
+    using System;
+    using System.Collections.Generic;
+
+    public class EvidenceNoteHistoryDataViewModelMapTransfer
```

Comment:
Could get away without this class. Just make DisplayEvidenceNoteHistoryData,

IMap<Ilist<EvidenceNoteHistoryData>, IList<EvidenceNoteHistoryViewModel>>